rules/integrations/aws/initial_access_signin_console_login_no_mfa.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2024/08/19"
+integration = ['aws']
+maturity = "production"
+updated_date = "2024/08/19"
+min_stack_comments = "ES|QL rule type in technical preview as of 8.13"
+min_stack_version = "8.13.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a federated user logs into the AWS Management Console without using multi-factor authentication (MFA). Federated users are typically given temporary credentials to access AWS services. If a federated user logs into the AWS Management Console without using MFA, it may indicate a security risk, as MFA adds an additional layer of security to the authentication process. This could also indicate the abuse of STS tokens to bypass MFA requirements.
```

Comment:
```suggestion
Identifies when a federated user logs into the AWS Management Console without using multi-factor authentication (MFA).
Federated users are typically given temporary credentials to access AWS services. If a federated user logs into the AWS
Management Console without using MFA, it may indicate a security risk, as MFA adds an additional layer of security to
the authentication process. This could also indicate the abuse of STS tokens to bypass MFA requirements.
```